eng/scripts/Language-Settings.ps1

```diff
@@ -152,4 +153,18 @@ function Publish-java-GithubIODocs ($DocLocation, $PublicArtifactLocation)
       }
     }
   }
+}
+
+function Generate-java-GithubIODocIndex () {
+  $metadata = Get-java-Metadata 
+  $filterOptions = @{
+    'Hide' = ''
+    'GroupId' = 'com.azure'
+  }
+  Generate-DocIndex -metadata $metadata -filterOptions $filterOptions -blobStorageUrl $BlobStorageUrl -blobDirectoryRegex "^java/(.*)/$" -titleRegex "Java" 
+}
+
+function Get-java-Metadata () {
```

Comment:
Given that this is the same for each language and they already define the $MetadataUri I think we can do this in a common `Get-PkgMetadata` function in one of the common scripts.

Reply:
I am putting the Get-CSVMetadata function into Generate-DocIndex currently

Reply:
Ok we should consider moving into another shared location because i'm pretty sure other places will need it as well.